src/ychaos/core/executor/common.py

```diff
@@ -22,15 +21,14 @@ class YChaosAnsibleResultCallback(CallbackBase, EventHook):
         This is a callback plugin which help performing actions as results from running the ansible playbook comes in.
         """
 
-        __hook_events__ = collections.defaultdict(
-            EventHook.CallableType(TaskResult)
-        ).fromkeys(
-            (
-                "on_target_unreachable",
-                "on_target_failed",
-                "on_target_passed",
-            )
+        events = (
+            "on_target_unreachable",
+            "on_target_failed",
+            "on_target_passed",
         )
+        __hook_events__ = dict()
+        for x in events:
+            __hook_events__[x] = EventHook.CallableType(TaskResult)
```

Comment:
```suggestion
        __hook_events__: Dict[str, Callable[[Any, ...], None] = collections.defaultdict(
            EventHook.CallableType(TaskResult)
        ).fromkeys(
            (
                "on_target_unreachable",
                "on_target_failed",
                "on_target_passed",
            )
        )
```

How about this? Reason is to avoid creating branches just to add type annotations

Reply:
mypy fails with this

```python3
collections.defaultdict(
            EventHook.CallableType(TaskResult)
        ).fromkeys(
            (
                "on_target_unreachable",
                "on_target_failed",
                "on_target_passed",
            )
        )
```
Has type Dict[str,None]